### DISCUSSION TITLE
All user can check user profile

### DISCUSSION THREAD

**Discussion by Pjaijai** [Pjaijai/Referalah#70] [投票 | Polls]
Currently, people cannot check other users' profile page.
What do you think we make the page dynamic from `/profile/page.ts` to `/profile/[slug]` something like that

If vote yes, lets discuss the detailed ???

cut-off: 13/10/2023 (Toronto time)

**Comment by 511234** [comment]
Currently there's too much information on referee / referer template.tsx, as a user I just want to glimpse a person's summary without knowing all of their experience. Besides, I find it troublesome switching my cursor between scrolling each of their description and scrolling the whole page. If I am really interested, I could visit their profile page to learn more.

But I also concern if we proceed to make profile page, it'd be very similar to Linkedin, so it gets me thinking what'd make our profile page differ from Linkedin - perhaps on `template.tsx`, we'd rename `description` to `summary`, which is set to a lower word limit, and on profile/[slug], we create  in `description`, and encourage people to be expressive and write in Cantonese?

**Reply by Pjaijai** [reply]
Yea agree. Thats my ideal approach to refine the UIUX.

**Comment by 511234** [comment]
Also @Pjaijai if you could clarify the differences between `post/referee` & `/referee`, and `post/referer` & `/referer`, coz they look fairly similar to me, and we as new collaborators shall avoid duplicating functions before we start working on features...

**Reply by Pjaijai** [reply]
For post, It's like placing job advertisements on the street. People reach out when they spot the ads.

For referee and referrer. Think of it as being at a party where people approach you to say hello.

**Comment by Pjaijai** [comment]
btw @511234  do you have any UXUI design experience ?

**Reply by 511234** [reply]
I just know how to write code based on designer's ideas, but I am not a designer myself. :(
